blocks/library/heading/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Internal dependencies
+ */
+import { registerBlock, query } from 'api';
+import Editable from 'components/editable';
+
+const { html, prop } = query;
+
+registerBlock( 'core/heading', {
+	title: wp.i18n.__( 'Heading' ),
+
+	icon: 'heading',
+
+	category: 'common',
+
+	attributes: {
+		content: html( 'h1,h2,h3,h4,h5,h6' ),
+		tag: prop( 'h1,h2,h3,h4,h5,h6', 'nodeName' ),
+		align: prop( 'h1,h2,h3,h4,h5,h6', 'style.textAlign' )
+	},
+
+	controls: [
+		...'123456'.split( '' ).map( ( level ) => ( {
+			icon: 'heading',
+			text: level,
+			title: wp.i18n.sprintf( wp.i18n.__( 'Heading %s' ), level ),
```

Comment:
>Sounds good too. Does it matter much? Sorry for my ignorance.

I'll not claim to be a localization expert, but perhaps as a hint to the translator that it's expected to be translated with a number vs. any arbitrary string could affect the translated text. Also prevents said arbitrary strings if we're very intentional that it's not to be expected (as a result of some potentially buggy code).